app/controllers/comments_cotroller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ActionController::Base
```

Comment:
- [ ] Kindly remove this controller. We only need posts and users controllers. Plus there is no route for this controller, which we also don't need.